boards/nordic/nrf7120pdk/board.cmake

```diff
@@ -15,6 +15,7 @@ if(CONFIG_TFM_FLASH_MERGED_BINARY)
   set_property(TARGET runners_yaml_props_target PROPERTY hex_file tfm_merged.hex)
 endif()
 
+include(${CMAKE_CURRENT_LIST_DIR}/../common/nrfutil_next.board.cmake)
 include(${ZEPHYR_BASE}/boards/common/nrfjprog.board.cmake)
 include(${ZEPHYR_BASE}/boards/common/nrfutil.board.cmake)
```

Comment:
does it make sense to have nrfutil here if nrfutil_next is a superset of nrfutil?